fecgov/fec-proxy
# Add redirects for /pdf files

Resolves #https://github.com/fecgov/fec-cms/issues/5292

Redirects files that were in the /pdf and /pdf/forms/ subdirectories. Includes some stray subdirectories with only one thing in them.

Attaching text file with just the redirects that were included. Many items in the 18.4 round were either moved into a future ticket or a take down ticket, so the usual spreadsheet is messy this time around.
[18-4-redirects.txt](https://github.com/fecgov/fec-proxy/files/8970850/18-4-redirects.txt)

To test:
- Check redirect code
- Check for alphabetizing with the other redirects.
